s3prl/downstream/asr/hf_hub_templates/model.py

```diff
@@ -0,0 +1,39 @@
+from s3prl.downstream.runner import Runner
```

Comment:
we provide a `hf_hub_templates` folder for each downstream task, which includes (minimally) `model.py` (the interface for the Hub) and `README.md` files. 

in this PR we include templates for ASR and SD and can add the others in separate PRs

Reply:
Cool! LGTM.